### PR TITLE
issue/gulp

### PR DESCRIPTION
**Pull request by tiffy74**
Updated AngularJS reference document by replacing all references to Grunt with gulp as Grunt was removed from Umbraco build process back in Summer 2017.

### CODE REVIEW

File: Development-Guidelines/building-angular-project.md [hunk 1]
```diff
@@ -93,3 +93,98 @@ Having Umbraco 7 UI as a separate project does indeed give us a bit more complex
 However the alternative would be to shove everything into the MSBuild process, making the entire thing inaccessible to a large number of frontend developers and with a clunkier and less up to date system.
 
 So see it as an additional powerful tool in your arsenal, once you see the power, you don't want to go back.
+# Working with the back office UI AngularJs project 
+
+## Overview
+Umbraco 7 has a slightly unorthodox project structure, compared to a normal asp.net project. This is by design, a choice from the beginning to embrace a much larger group than "just" the developers who know how to use Visual Studio. 
+
+As a result, the Umbraco UI is not a Visual Studio project, but simply a collection of folders and files, following certain conventions, and a small configuration file called `gruntfile` - we will get to the Grunt part in a moment.
```
[the remark this answers is not in the document]
Comment (marcemarc) [reply Development-Guidelines/building-angular-project.md]: Hi @tiffy74 - cool, If you just make the changes in your fork branch, and push, the PR will be magically updated!

**Comment by tiffy74** [comment]
Have now swapped out all references to Grunt in favour of gulp :)

**Comment by tiffy74** [comment]
Yes I can see. Humph. Will fix it asap

**Comment by jmayntzhusen** [comment]
Hey @tiffy74 , the bad merge has indeed been fixed, thanks a lot! 😉

I must admit I don't know enough about the differences in gulp and grunt to know if the console commands are the exact same, if they are - great then we can merge. Will try to test it out with your updated documentation today and get back to you!

File: Development-Guidelines/building-angular-project.md [hunk 2]
```diff
@@ -16,18 +16,18 @@ Umbraco 7 needs a couple of things to run:
 ### Node.js 
 To compile and run the UI project you need Node.js installed, you can get that at [https://nodejs.org](nodejs.org) for both Windows and OSX.
 
-### Grunt
-When you have Node.js installed, you need to install Grunt. Grunt is a simple JavaScript task runner, basically like NAnt, MSBuild or any traditional build system [https://gruntjs.com](more about grunt here).
+### gulp
+When you have Node.js installed, you need to install gulp. gulp is a simple JavaScript task runner, basically like NAnt, MSBuild or any traditional build system [https://gulpjs.com](more about gulp here).
```
Comment (jmayntzhusen) [inline Development-Guidelines/building-angular-project.md]: [https://gulpjs.com](more about gulp here)
should be 
[more about gulp here](https://gulpjs.com), don't know why that was never caught in the original file 😄

Comment (jmayntzhusen) [reply Development-Guidelines/building-angular-project.md]: Would also be great if you capitalized Gulp - if nowhere else then in the title and after a period!

Comment (tiffy74) [reply Development-Guidelines/building-angular-project.md]: Hiya thanks for taking a look.  I've fixed the references to the URL link.  Thanks for spotting it!

In terms of capitalisation - I decided to keep the lower case as this is how it seems to be referred to in their own documentation.  Similarly on the wiki regarding gulp is uses lower case g even within titles whereas the g in Grunt is capitalised.  So I concluded that gulp should always be lower case g.  However if you disagree then I would be happy to amend it as it is not something I'm going to be too precious about.

In terms of the commands - I did check the syntax from the gulp documentation.  It does indeed seem extremely similar.  However really appreciate it if you are going to test it within the context of Umbraco.  I did run the commands myself before I requested the PR but it always helps to have things verified :)

Once again - thanks.  Almost there!

**Comment by jmayntzhusen** [comment]
Hey @tiffy74 ,

You're right, all the commands are working and it seems be in order now, thanks for helping out with the Umbraco Documentation #h5yr! 😄

**Comment by dampee** [comment]
Well done @tiffy74   Thanks a lot for your effort!  H5YR!

**Pull request merged**